client/my-sites/patterns/components/header/style.scss

```
@@ -0,0 +1,74 @@
+
+@import "@automattic/components/src/styles/typography";
+@import "@wordpress/base-styles/breakpoints.scss";
+
+.patterns-header {
+	background: #1d2327;
+	padding: 0 26px;
+
+	.patterns-header--inner {
+		padding: 105px 0 64px 84px;
+		background: url(./images/background.svg) no-repeat right bottom;
+		color: var(--studio-white);
+
+		h1 {
+			font-family: $font-recoleta;
+			font-size: rem(70px);
+			line-height: 1;
+			letter-spacing: 0.2px;
+			width: 600px;
+			padding-bottom: 24px;
+		}
+
+		.patterns-header--description {
+			font-family: $font-sf-pro-display;
+			font-size: rem(18px);
+			letter-spacing: 0.4px;
+			line-height: 1.4;
+			width: 450px;
+			padding-bottom: 32px;
+		}
+
+		.patterns-header--search-input {
+			border-radius: 4px;
+			width: 450px;
+			height: 56px;
+			overflow: hidden;
+			margin-bottom: 0;
+
+			input[type="search"] {
+				font-family: $font-sf-pro-text;
+				letter-spacing: -0.32px;
+
+				&::placeholder {
+					color: #01283d;
+				}
+			}
+		}
+	}
+
+	@media ( max-width: $break-huge ) {
```

```suggestion
	@media ( max-width: $break-wide ) {
```
Not sure if `$break-wide` is the right choice here (maybe we could even go with something slightly smaller), but `$break-huge` definitely feels unnecessarily large.

The padding for `.patterns-header--inner` still needs to match `.patterns-section--body`, but I'm primarily thinking about the switch from `background: url(…)` to `background: none`

---

>Not sure if $break-wide is the right choice here (maybe we could even go with something slightly smaller), but $break-huge definitely feels unnecessarily large.

I don't have strong preference, it's all personal, so I am totally ok to go with `$break-wide`, addressed.

>The padding for .patterns-header--inner still needs to match .patterns-section--body, but I'm primarily thinking about the switch from background: url(…) to background: none

Yep, I had totally the same thoughts.